conf/base.config

```diff
@@ -15,8 +15,8 @@ process {
   memory = { check_max( 8.GB * task.attempt, 'memory' ) }
   time = { check_max( 4.h * task.attempt, 'time' ) }
 
-  errorStrategy = { task.attempt <= 10 ? 'retry' : 'ignore' }
-  maxRetries = 10
+  errorStrategy = { task.attempt <= 30 ? 'retry' : 'ignore' }
+  maxRetries = 30
```

Comment:
I was initially greedy and allocated all of the machine resources, and got this error:

> Relevant CloudOS job link: https://cloudos.lifebit.ai/public/jobs/5eb2dad91795470103dbfe38

```console
/tmp/nxf-8406753682392313333/.command.stub: fork: Cannot allocate memory
```